deepface/commons/functions.py

```diff
@@ -94,6 +94,14 @@ def load_image(img):
     if type(img).__module__ == np.__name__:
         return img, None
 
+    try:
+        # Test whether img is a Python3's Path. If hit, tranform to str to let following logic work.
+        from pathlib import Path
```

Comment:
This is already imported in line 3: https://github.com/serengil/deepface/blob/master/deepface/commons/functions.py#L3C21-L3C25

This causes linting error in github actions:

************* Module deepface.commons.functions
deepface/commons/functions.py:99:8: W0621: Redefining name 'Path' from outer scope (line 3) (redefined-outer-name)
deepface/commons/functions.py:99:8: W0404: Reimport 'Path' (imported line 3) (reimported)